phalcon/Http/Request.zep

```diff
@@ -763,6 +765,8 @@ class Request extends AbstractInjectionAware implements RequestInterface, Reques
         bool notAllowEmpty = false,
         bool noRecursive = false
     ) -> var {
+        let _POST = this->getPostData(_POST);
```

Comment:
I am not sure we should be manipulating the $_POST array here. Shouldn't we just use the getPostData(_POST) in the getHelper? Same comment for below.

Reply:
What I mean is that the request should be processing data that it got and that data has to be immutable. If the $_POST changes it should change in the next request not the current one mid way so to speak.